cli/cmd/superuser/autoinvite/create.go

```diff
@@ -0,0 +1,53 @@
+package autoinvite
+
+import (
+	"fmt"
+
+	"github.com/rilldata/rill/cli/pkg/cmdutil"
+	"github.com/rilldata/rill/cli/pkg/config"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
+	"github.com/spf13/cobra"
+)
+
+func CreateCmd(cfg *config.Config) *cobra.Command {
+	createCmd := &cobra.Command{
+		Use:    "create <org> <domain> <role>",
+		Args:   cobra.ExactArgs(3),
+		Short:  "Autoinvite users from a domain",
+		Hidden: true,
```

Comment:
Remove `Hidden` here and put it on the `sudo` command instead (then doing `rill sudo autoinvite -h` will show the available commands)